crowdfunding_compassion/models/crowdfunding_participant.py

```diff
@@ -18,9 +18,9 @@ class CrowdfundingParticipant(models.Model):
         "res.partner", string="Partner", required=True, index=True, ondelete="cascade"
     )
     personal_motivation = fields.Text()
-    product_number_goal = fields.Integer(default=1)
+    product_number_goal = fields.Integer(default=0)
```

Comment:
This will make an error when displaying a goal_progress progress bar, as product_number_goal is used as "diviseur" and if == 0, it will crash

Reply:
Then we should fix the progress bar because projects can have 0 product goal and only focus on sponsorships.

Reply:
@SylvainLosey should maybe refactor it

Reply:
> This will make an error when displaying a goal_progress progress bar, as product_number_goal is used as "diviseur" and if == 0, it will crash

Where is that ? I don't see `product_number_goal` used as a divider. 

Also if there is only sponsorships as goal and `product_number_goal` is 0, then `product_id` (the fund) would be null and the progress bar wouldn't be displayed

Reply:
If there is a situation where `product_id` is set but `product_number_goal` is 0, then we should just modify `goal_progress` a bit:

```<t t-set="progress_percentage" t-value="goal_reached / goal_objective * 100 if goal_objective else 0"/>```

Reply:
I have made the corrections.